client/src/theme/nativebase.js

```diff
@@ -5,6 +5,15 @@ import colors from './colors'
 const theme = extendTheme({
   colors,
   components: {
+    Input: {
+      baseStyle: {
+        size: '20',
+        //lineHeight: "xl",
+        selectionColor: 'coolGray.800',
+      },
```

Comment:
Can you make it so that the border color of the input field is coolGray.800 when not selected, and blue.medium (the 7L medium blue color) when selected? That way, the user can tell which field they are editing. Make sure that the Input and TextArea both have this style overriden.